scss/_dropdown.scss

```diff
@@ -54,8 +54,9 @@
 // The dropdown menu
 .dropdown-menu {
   // scss-docs-start dropdown-css-vars
+  --#{$prefix}dropdown-zindex: #{$zindex-dropdown}; // Boosted mod
```

Comment:
This change might be on Bs side too

Reply:
Yes, IMO it could be integrated in Bootstrap.

Reply:
Done in https://github.com/twbs/bootstrap/pull/37021